openedx/features/content_type_gating/tests/test_access.py

```diff
@@ -94,6 +97,14 @@ def setUpClass(cls):
             has_score=False,
         )
 
+        # add ungraded problem for xblock_handler test
+        cls.ungraded_problem = ItemFactory.create(
+            parent=cls.blocks_dict['vertical'],
+            category='problem',
+            display_name='problem_2',
```

Comment:
could you name the two problems ungraded_problem and graded_problem for clarity?